### ISSUE TITLE
Petition - Still prompted to `Sign Now` after signing Pledge

### ISSUE THREAD

**Issue by 0xDaedalus** [tahowallet/taho.xyz#85]
### What Happened
Right after signing the pledge (or after visiting the site with a wallet that has already signed the pledge) there is still a prompt to `Sign Now`.

<img width="382" alt="image" src="https://user-images.githubusercontent.com/94649004/186005702-da88230b-a75a-4802-b179-95bc3cd1e41c.png">

### Expected
Something indicating that the user has already signed the petition

**Comment by mr-michael** [comment]
Yeah unf it's an "is what it is" situation. If we want to do a fast-follow (next week) with a "you're connected and you've already signed state" I think that's a solid idea, cc @VladUXUI for thoughts
